src/test/java/uk/gov/register/functional/store/postgres/RegisterImplTransactionalFunctionalTest.java

```diff
@@ -72,6 +72,31 @@ public void tearDown() throws Exception {
         dbi.close(handle);
     }
 
+    @Test
+    public void getItemUsingNewHash() {
```

Comment:
Is it worth a `TODO` here to use the new hashing algorithm once implemented?

Reply:
I think this is ok actually... for the purposes of the test it doesn't matter what the hashes are, just that you can look up items by either one.

When I implement objecthash in `Item` I'll add separate tests for that.